doc/classes/SkeletonModification3DFABRIK.xml

```diff
@@ -117,6 +123,13 @@
 				Sets the magenet position to [code]magnet_position[/code] for the joint at [code]joint_idx[/code]. The magnet position is used to nudge the joint in that direction when solving, which gives some control over how that joint will bend when being solved.
```

Comment:
magnet? Typo.

Reply:
That was not the only typo I saw while working on those files, but I am not the author of that section and I have to put a limit to how far I want to fix things.
Corrections outside the scope of what I added would be better dealt with a different PR in my opinion.

Reply:
That's fine.